### ISSUE TITLE
Create the Join Us page with a contact form

### ISSUE THREAD

**Issue by mutharasus** [AccessibleAI-edu/accessibleai-edu#109]

**Comment by MarijaIgnj** [comment]
@mutharasus Can we publish the call for volunteers on this page? I am attaching the PDF

[Call for volunteers AAI (250 × 320mm) (250 × 350mm) (3).pdf](https://github.com/AccessibleAI-edu/accessibleai-edu/files/13378696/Call.for.volunteers.AAI.250.x.320mm.250.x.350mm.3.pdf)

**Comment by mutharasus** [comment]
@MarijaIgnj Can you please check the pull request link to see if this looks as expected. I basically placed the PDF in the page as an object (without any of the PDF tools). For the links to work it needs to be a PDF file and not a JPG like we have in infographic pages. Also border has to be grey instead of white as that is the PDF container that shows the scroll bar etc and the color gets set by the browser.

https://pr-113.d241og4j1y7lf6.amplifyapp.com/join-us.html

**Comment by MarijaIgnj** [comment]
@mutharasus Yes, that is fine. Can you please put small U in Joun Us instead of the capital. It should be generally changed everywhere, this way it looks like Chat GPT made the website :D

**Comment by MarijaIgnj** [comment]
@mutharasus The join us page need to be properly formatted.
